documentation/guide-jpa-query.asciidoc

```diff
@@ -114,7 +114,29 @@ The `orderBy` method is used to sort the query results according to certain crit
 
 For spring, devon4j provides another approach that you can use for your Spring applications to implement Querydsl logic without having to use these metaclasses. An example can be found link:spring/guide-querydsl-spring.asciidoc[here].
 
+== Native Queries
+Spring Data supports the use of _native queries_. Native queries use simple native SQL syntax that is not parsed in JPQL. This allows you to use all the features that your database supports.
+The downside to this is that database portability is lost because there is no abstraction layer and the query may not work with another database because it may use a different syntax.
```

Comment:
Suggestion:
The downside to this is that database portability is lost due to the absence of an abstraction layer. Additionally, the queries may not work with another database because it may use a different syntax.